ui/src/pages/Demo.jsx

```diff
@@ -51,9 +52,10 @@ class DemoPage extends React.Component {
                 <h1>Helper App</h1>
                 <ButtonPrimary />
                 <ButtonSecondary />
-                <ProfileCard />
+                {/* <ProfileCard /> */}
```

Comment:
We need to address the error message that this component was throwing you - we can do this together when we meet to deploy.